app/reducers/quotes.js

```diff
@@ -80,12 +83,14 @@ export const selectQuoteById = createSelector(
 export const selectSortedQuotes = createSelector(
   selectQuotes,
   (state, props) => ({ filter: props.filter }),
```

Comment:
Maybe remove this since it's not used anymore

Reply:
Yeee, will do :ok_hand: